recipe/meta.yaml

```
@@ -1,57 +1,90 @@
 {% set name = "cassandra-driver" %}
-{% set version = "3.25.0" %}
-{% set sha256 = "8ad7d7c090eb1cac6110b3bfc1fd2d334ac62f415aac09350ebb8d241b7aa7ee" %}
+{% set version = "3.29.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://github.com/datastax/python-driver/archive/refs/tags/{{ version }}.tar.gz
+  sha256: aa3be5396e05b395c178091656a329daba23b0d4dd69b8d076090157f86e6d13
 
 build:
+  skip: true  # [py<38 or py>312]
   number: 0
-  script: python setup.py build build_ext -I${PREFIX}/include -L${PREFIX}/lib install --single-version-externally-managed --record record.txt  # [not win]
-  script: python setup.py install --no-cython --single-version-externally-managed --record record.txt  # [win]
-  skip: true  # [py2k]
+  script: {{ PYTHON }} setup.py build build_ext -I${PREFIX}/include -L${PREFIX}/lib install --single-version-externally-managed --record record.txt  # [not win]
```

Try to build with **pip install** instead of **setup**
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```